pldoc/static/sass/_utilities.scss

```diff
@@ -34,6 +36,10 @@ $pldoc-ui-toolkit-color:            rgb(255, 222, 32);
     }
 }
 
+%grid-wrapper {
```

Comment:
Nit: this seems like it should go in `_layouts.scss` instead.  Seems layout-like?